server/templates/cloud_welcome_email.html

```
@@ -30,19 +26,19 @@
                                             {{.Props.Title}}
                                         </p>
                                         <p
-                                            style="font-family: Open Sans, Arial; font-style: normal; font-weight: 400; font-size: 16px; line-height: 24px; color: #3D3C40;">
+                                            style="font-family: sans-serif; font-style: normal; font-weight: 400; font-size: 16px; line-height: 24px; color: #3D3C40;">
                                             {{.Props.Info}} <a href="{{.Props.WorkSpacePath}}" target="_blank"
                                                 rel="noopener noreferrer"
                                                 style="text-decoration: none; color: #6DA4EB;">{{.Props.DNS}}.</a>
                                             {{.Props.Info2}}
                                         </p>
                                         <p style="text-align: center; margin-top: 30px; margin-bottom: 30px;">
                                             <a href="{{.Props.WorkSpacePath}}" target="_blank" rel="noopener noreferrer"
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </a> ], start tag match failed [ <a href="{{.Props.WorkSpacePath}}" target="_blank" rel="noopener noreferrer"
                                                style="padding: 14px 24px; position: static; background: `#1C58D9`; border-radius: 4px; margin: 0px 150px; font-family: sans-serif; font-style: normal; font-weight: 600; font-size: 16px; line-height: 18px; color: `#FFFFFF`; text-decoration: none;"> ] on line 40.

(tag-pair)